src/styles/sass/img/contents.svg

```diff
@@ -0,0 +1,7 @@
+<svg viewBox="0 0 24 24">
```

Comment:
why are there svg files as well as the stuff in IFrameNavigator? are they both used?